Makefile

```diff
@@ -138,13 +138,19 @@ generate-chart: crossplane
 # where the operator is deployed. See https://github.com/operator-framework/operator-lifecycle-manager/issues/1361
 # and https://github.com/operator-framework/operator-lifecycle-manager/issues/2039
 
-olm: $(HELM) $(OLMBUNDLE)
+olm.build: $(HELM) $(OLMBUNDLE)
 	@$(INFO) Generating OLM bundle
 	@$(HELM) -n upbound-system template $(HELM_CHARTS_DIR)/$(PACKAGE_NAME) --set upbound.controlPlane.permission=edit > $(WORK_DIR)/olm.yaml
 	@$(SED_CMD) 's|RELEASE-NAME|$(PROJECT_NAME)|g' $(WORK_DIR)/olm.yaml
 	@rm -rf $(OLM_DIR)/bundle
 	@cat $(WORK_DIR)/olm.yaml | $(OLMBUNDLE) --version $(HELM_CHART_VERSION) --chart-file-path $(HELM_CHARTS_DIR)/$(PACKAGE_NAME)/Chart.yaml --extra-resources-dir $(CRDS_DIR) --output-dir $(OLM_DIR)
 
+olm.artifacts: olm.build
+	@mkdir -p $(abspath $(OUTPUT_DIR)/olm/$(VERSION))
+	@cp -r $(OLM_DIR)/bundle/. $(abspath $(OUTPUT_DIR)/olm/$(VERSION))
```

Comment:
I think it might make more sense to put the contents here into a tarball to publish.

Reply:
Does build submodule take care of uploading whatever we put under `$(OUTPUT_DIR)`?

Reply:
It does 👍 You can see examples of this in `up` where I put the debs and rpm in the output dir https://cli.upbound.io/p?prefix=stable/v0.2.0/

Reply:
Got it. Tarball makes sense to me, too. Folder downloading may not be as straightforward.